# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1499465852 14400
#      Fri Jul 07 18:17:32 2017 -0400
# Node ID 6921d8458203ba5d6a2dc2addf9c5e7d9dea5b60
# Parent  6e86d3b5a0632db625ee9e3dc5ce56764068b265
move comment-list.h and comment-list.cc to libinterp/parse-tree directory

* libinterp/parse-tree/comment-list.cc,
libinterp/parse-tree/comment-list.h: Move here from libinterp/corefcn
directory.
* libinterp/corefcn/module.mk, libinterp/parse-tree/module.mk:
Update.

diff --git a/libinterp/corefcn/module.mk b/libinterp/corefcn/module.mk
--- a/libinterp/corefcn/module.mk
+++ b/libinterp/corefcn/module.mk
@@ -17,17 +17,16 @@ OPT_HANDLERS = \
 DIRSTAMP_FILES += %reldir%/$(octave_dirstamp)
 
 COREFCN_INC = \
   %reldir%/base-text-renderer.h \
   %reldir%/Cell.h \
   %reldir%/c-file-ptr-stream.h \
   %reldir%/call-stack.h \
   %reldir%/cdisplay.h \
-  %reldir%/comment-list.h \
   %reldir%/data.h \
   %reldir%/defaults.h \
   %reldir%/defun-dld.h \
   %reldir%/defun-int.h \
   %reldir%/defun.h \
   %reldir%/dirfns.h \
   %reldir%/display.h \
   %reldir%/dynamic-ld.h \
@@ -119,17 +118,16 @@ COREFCN_SRC = \
   %reldir%/bitfcns.cc \
   %reldir%/bsxfun.cc \
   %reldir%/c-file-ptr-stream.cc \
   %reldir%/call-stack.cc \
   %reldir%/cdisplay.c \
   %reldir%/cellfun.cc \
   %reldir%/colloc.cc \
   %reldir%/coct-hdf5-types.c \
-  %reldir%/comment-list.cc \
   %reldir%/conv2.cc \
   %reldir%/daspk.cc \
   %reldir%/dasrt.cc \
   %reldir%/dassl.cc \
   %reldir%/data.cc \
   %reldir%/debug.cc \
   %reldir%/defaults.cc \
   %reldir%/defun.cc \
diff --git a/libinterp/corefcn/comment-list.cc b/libinterp/parse-tree/comment-list.cc
rename from libinterp/corefcn/comment-list.cc
rename to libinterp/parse-tree/comment-list.cc
diff --git a/libinterp/corefcn/comment-list.h b/libinterp/parse-tree/comment-list.h
rename from libinterp/corefcn/comment-list.h
rename to libinterp/parse-tree/comment-list.h
diff --git a/libinterp/parse-tree/module.mk b/libinterp/parse-tree/module.mk
--- a/libinterp/parse-tree/module.mk
+++ b/libinterp/parse-tree/module.mk
@@ -1,10 +1,11 @@
 PARSE_TREE_INC = \
   %reldir%/bp-table.h \
+  %reldir%/comment-list.h \
   %reldir%/jit-ir.h \
   %reldir%/jit-typeinfo.h \
   %reldir%/jit-util.h \
   %reldir%/lex.h \
   %reldir%/parse.h \
   %reldir%/pt-all.h \
   %reldir%/pt-arg-list.h \
   %reldir%/pt-array-list.h \
@@ -41,16 +42,17 @@ PARSE_TREE_INC = \
   %reldir%/token.h
 
 
 ## oct-gperf.h and oct-parse.h are in the SRC list so that they will
 ## be distributed but not installed.
 
 PARSE_TREE_SRC = \
   %reldir%/bp-table.cc \
+  %reldir%/comment-list.cc \
   %reldir%/jit-ir.cc \
   %reldir%/jit-typeinfo.cc \
   %reldir%/jit-util.cc \
   %reldir%/lex.ll \
   %reldir%/oct-gperf.h \
   %reldir%/oct-parse.h \
   %reldir%/oct-parse.yy \
   %reldir%/pt-arg-list.cc \
